App/Containers/ThreadsList.js

```diff
@@ -46,7 +46,9 @@ class ThreadsList extends React.PureComponent {
       </TextileHeaderButtons>
     )
     const headerTitle = (
-      <Image style={navStyles.headerLogo} source={require('../SB/views/ThreadsList/statics/logo.png')} />
+      <View style={navStyles.headerCenter}>
```

Comment:
android also likes to left align all our headers. so a few fixes in here for those

Reply:
See my other comment about this.